lib/css/CssGenerator.js

```diff
@@ -45,82 +48,91 @@ class CssGenerator extends Generator {
+					exports: new Map()
+				};
+
+				generateContext.runtimeRequirements.add(RuntimeGlobals.hasCssModules);
+
+				let chunkInitFragments;
+				/** @type {DependencyTemplateContext} */
+				const templateContext = {
+					runtimeTemplate: generateContext.runtimeTemplate,
+					dependencyTemplates: generateContext.dependencyTemplates,
+					moduleGraph: generateContext.moduleGraph,
+					chunkGraph: generateContext.chunkGraph,
+					module,
+					runtime: generateContext.runtime,
+					runtimeRequirements: generateContext.runtimeRequirements,
+					concatenationScope: generateContext.concatenationScope,
+					codeGenerationResults: generateContext.codeGenerationResults,
+					initFragments,
+					cssExportsData,
+					get chunkInitFragments() {
+						if (!chunkInitFragments) {
+							const data = generateContext.getData();
+							chunkInitFragments = data.get("chunkInitFragments");
+							if (!chunkInitFragments) {
+								chunkInitFragments = [];
+								data.set("chunkInitFragments", chunkInitFragments);
+							}
+						}
+
+						return chunkInitFragments;
+					}
+				};
+
+				/**
+				 * @param {Dependency} dependency dependency
+				 */
+				const handleDependency = dependency => {
+					const constructor =
+						/** @type {new (...args: any[]) => Dependency} */ (
+							dependency.constructor
+						);
+					const template = generateContext.dependencyTemplates.get(constructor);
+					if (!template) {
+						throw new Error(
+							`No template for dependency: ${dependency.constructor.name}`
+						);
+					}
+
+					template.apply(dependency, source, templateContext);
+				};
+				for (const dependency of module.dependencies) {
+					handleDependency(dependency);
+				}
+				if (module.presentationalDependencies !== undefined) {
+					for (const dependency of module.presentationalDependencies) {
+						handleDependency(dependency);
 					}
 				}
 
-				return chunkInitFragments;
-			}
-		};
-
-		/**
-		 * @param {Dependency} dependency dependency
-		 */
-		const handleDependency = dependency => {
-			const constructor = /** @type {new (...args: any[]) => Dependency} */ (
-				dependency.constructor
-			);
-			const template = generateContext.dependencyTemplates.get(constructor);
-			if (!template) {
-				throw new Error(
-					`No template for dependency: ${dependency.constructor.name}`
-				);
+				return InitFragment.addToSource(source, initFragments, generateContext);
 			}
-
-			template.apply(dependency, source, templateContext);
-		};
-		for (const dependency of module.dependencies) {
-			handleDependency(dependency);
-		}
-		if (module.presentationalDependencies !== undefined) {
-			for (const dependency of module.presentationalDependencies) {
-				handleDependency(dependency);
+			case "javascript": {
+				const assetGenerator = new AssetGenerator();
+				return assetGenerator.generate(module, generateContext);
 			}
 		}
-
-		const data = generateContext.getData();
-		data.set("css-exports", cssExportsData);
```

Comment:
I don't think we should remove it from here

Reply:
yes, these codes were accidentally removed